### PR TITLE
Perf fixes

### LINKED ISSUE #2800: Stop toggling button state onActivated
According to kmag, webextension buttons have per-tab state, so setting the button enabled or disabled on update should be enough. Updating the icon on activated may be causing performance issues.

### LINKED ISSUE #2813: Remove WebExtension startup delay
Another performance tweak requested by kmag to address https://bugzilla.mozilla.org/show_bug.cgi?id=1361792.

### PR DESCRIPTION
**Pull request by 6a68**
Fixes related to the Talos regressions tracked by https://bugzilla.mozilla.org/show_bug.cgi?id=1361792.

These changes have already been reviewed by kmag: https://reviewboard.mozilla.org/r/136348/diff/2#index_header

Fixes #2800, fixes #2813.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+## Version 6.6.1
+
+* Correctly set Sentry URL when exporting to mozilla-central ([#2782](https://github.com/mozilla-services/screenshots/issues/2782))
+
+This version includes two changes that addressed Talos performance regressions discussed in [https://bugzil.la/1361792](https://bugzil.la/1361792):
+
+* Do not update toolbar button state inside tab 'activated' event handler ([#2800](https://github.com/mozilla-services/screenshots/issues/2800))
```
Comment (dannycoates) [inline CHANGELOG.md]: I'm confused about this... the `onActivated` handler still exists in main.js. The change in this commit to `onUpdated` makes sense but seems unrelated.

Comment (6a68) [inline CHANGELOG.md]: @dannycoates good catch, I missed one commit when pushing to this branch last night :-P

Comment (6a68) [inline CHANGELOG.md]: looks like it was badly performed git surgery. rebased and squashed, the missing bits should be present now :+1:

**Comment by 6a68** [comment]
whoops, I'll rebase in a bit. have another perf fix to add anyway

**Comment by 6a68** [comment]
Updated to add today's changes, now fixes #2824.

**Comment by 6a68** [comment]
🍻
